changelog.d/6317.misc

```diff
@@ -0,0 +1 @@
+Add optional python dependencies and dependant binary libraries to snapcraft packaging
```

Comment:
```suggestion
Add optional python dependencies and dependant binary libraries to snapcraft packaging.
```

Changelog entries need to end in a `.` or `!`.

Reply:
Done, and thanks